### ISSUE TITLE
[Bug]: Multiple groups move can't be undone

### ISSUE THREAD

**Issue by alessandroperilli** [Comfy-Org/ComfyUI_frontend#1327]
### Frontend Version

1.3.24

### Expected Behavior

After selecting multiple groups with CTRL and moving them all together with SHIFT, the groups are in a new position. I would expect that an Undo command would return them to their original position.

### Actual Behavior

The undo is ignored.

### Steps to Reproduce

Select, move, release the mouse, undo (either with shortcuts or with menu entry).

### Debug Logs

```powershell
not relevant
```

### Browser Logs

None produced after undo invocation.

### Setting JSON

Does it matter?

### What browsers do you use to access the UI ?

Other

### Other

Vivaldi on macOS

**Comment by huchenlei** [comment]
I cannot reproduce this locally.

https://github.com/user-attachments/assets/6b69bf99-eb3b-43bd-ad3a-cb7e232a4034

**Comment by alessandroperilli** [comment]
@huchenlei It's not the first time that the Undo command doesn't work for me. But, in the past, it has always been temporary and I assumed it was due to a glitch in the latest version of the front end which would go away with the next update. 
Perhaps, it's not the case. Perhaps, in my system, the undo is not always working consistently. What can I do to test this more extensively and give you useful debug info?

**Comment by LukeG89** [comment]
@alessandroperilli Could it be that all your issues you posted are browser-related? Have you tried another browser?

**Comment by alessandroperilli** [comment]
@LukeG89 That would not explain why I have none of the other two issues I posted today with version 1.3.8. 
But, for the sake of research, I just tested with Safari, and I see the same problems described so far. I'll update the other two open issues with this extra information.

In doing this test for you, I realized what's the problem for this specific issue: if you move the group, click somewhere else to deselect, and then you undo, ComfyUI first undoes the deselection and then undoes the group move.

Now, because the multi-group selection is a very subtle visual effect at the zoom level I normally operate with APW, it seems like nothing has happened. There's no other visual cue about what has been undone, so I had the impression that the undo was not working. 

I just need to issue the command twice, or avoid deselecting the multi-group. 

Sorry about this!
